### PR TITLE
qa/*/test_envlibrados_for_rocksdb.sh: install libarchive-3.3.3

### PR DESCRIPTION
**Pull request by neha-ojha**
To workaround the libarchive dependency issue seen with centos 8, which
has been causing consistent failures like

```
2021-06-04T04:52:51.147 INFO:tasks.workunit.client.0.smithi071.stdout:Installed:
2021-06-04T04:52:51.148 INFO:tasks.workunit.client.0.smithi071.stdout:  cmake-3.18.2-9.el8.x86_64                 cmake-data-3.18.2-9.el8.noarch
...
2021-06-04T04:52:57.554 INFO:tasks.workunit.client.0.smithi071.stderr:+ cmake -DCMAKE_BUILD_TYPE=Debug -DWITH_TESTS=ON -DWITH_LIBRADOS=ON -DWITH_SNAPPY=ON -DWITH_GFLAGS=OFF -DFAIL_ON_WARNINGS=OFF ..
2021-06-04T04:52:57.579 DEBUG:teuthology.orchestra.run:got remote process result: 127
2021-06-04T04:52:57.580 INFO:tasks.workunit.client.0.smithi071.stderr:cmake: symbol lookup error: cmake: undefined symbol: archive_write_add_filter_zstd
```

Fixes: https://tracker.ceph.com/issues/51101
Signed-off-by: Neha Ojha <nojha@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by neha-ojha** [comment]
Before: https://pulpito.ceph.com/nojha-2021-07-12_21:34:59-rados:singleton-master-distro-basic-smithi/6266805/
After: https://pulpito.ceph.com/nojha-2021-07-12_22:32:46-rados:singleton-master-distro-basic-smithi/

File: qa/workunits/rados/test_envlibrados_for_rocksdb.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ case $(distro_id) in
                 sudo subscription-manager repos --enable "codeready-builder-for-rhel-8-x86_64-rpms"
                 ;;
         esac
-        install git gcc-c++.x86_64 snappy-devel zlib zlib-devel bzip2 bzip2-devel libradospp-devel.x86_64 cmake
+        install git gcc-c++.x86_64 snappy-devel zlib zlib-devel bzip2 bzip2-devel libradospp-devel.x86_64 cmake libarchive
```
Comment (tchaikov) [inline qa/workunits/rados/test_envlibrados_for_rocksdb.sh:36]:
```suggestion
        install git gcc-c++.x86_64 snappy-devel zlib zlib-devel bzip2 bzip2-devel libradospp-devel.x86_64 cmake libarchive-3.3.3
```

as libarchive could be already installed on the building host, and that was the case in https://tracker.ceph.com/issues/51101. to ensure we have access to a libarchive with zstd support, might want to specify the version as well.

**Review by tchaikov — APPROVED** [review]
another solution is to update the centos8-stream image so it contains an updated libarchive package.

File: qa/workunits/rados/test_envlibrados_for_rocksdb.sh (hunk 1 above)
Comment (neha-ojha) [reply qa/workunits/rados/test_envlibrados_for_rocksdb.sh]: sure, done! https://pulpito.ceph.com/nojha-2021-07-12_22:32:46-rados:singleton-master-distro-basic-smithi/

**Comment by neha-ojha** [comment]
> Before: https://pulpito.ceph.com/nojha-2021-07-12_21:34:59-rados:singleton-master-distro-basic-smithi/6266805/
> After: https://pulpito.ceph.com/nojha-2021-07-12_22:32:46-rados:singleton-master-distro-basic-smithi/

New test results: https://pulpito.ceph.com/nojha-2021-07-12_22:32:46-rados:singleton-master-distro-basic-smithi/

**Review by tchaikov — APPROVED** [review]

**Comment by neha-ojha** [comment]
jenkins test make check

**Comment by neha-ojha** [comment]
jenkins test make check

**Pull request merged**